src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -273,37 +269,47 @@ protected boolean prepareRecipeDistinct(Recipe recipe) {
                 getMaxParallelVoltage(),
                 getParallelLimit());
 
-        if (recipe != null && setupAndConsumeRecipeInputs(recipe, currentDistinctInputBus)) {
-            setupRecipe(recipe);
-            return true;
+        if (recipe != null) {
+            recipe = setupAndConsumeRecipeInputs(recipe, currentDistinctInputBus);
+            if (recipe != null) {
+                setupRecipe(recipe);
+                return true;
+            }
         }
 
         return false;
     }
 
     @Override
-    protected void modifyOverclockPre(int @NotNull [] values, @NotNull IRecipePropertyStorage storage) {
-        super.modifyOverclockPre(values, storage);
+    protected void modifyOverclockPre(@NotNull OCParams ocParams, @NotNull IRecipePropertyStorage storage) {
+        super.modifyOverclockPre(ocParams, storage);
 
         // apply maintenance bonuses
         Tuple<Integer, Double> maintenanceValues = getMaintenanceValues();
 
         // duration bonus
         if (maintenanceValues.getSecond() != 1.0) {
-            values[1] = (int) Math.round(values[1] * maintenanceValues.getSecond());
+            ocParams.setDuration((int) Math.round(ocParams.duration() * maintenanceValues.getSecond()));
         }
     }
 
     @Override
-    protected void modifyOverclockPost(int[] overclockResults, @NotNull IRecipePropertyStorage storage) {
-        super.modifyOverclockPost(overclockResults, storage);
+    protected void runOverclockingLogic(@NotNull OCParams ocParams, @NotNull OCResult ocResult,
+                                        @NotNull IRecipePropertyStorage propertyStorage, long maxVoltage) {
+        subTickParallelOC(ocParams, ocResult, maxVoltage, getOverclockingDurationFactor(),
```

Comment:
I thought we did not want our multis in base CEu to sub-tick parallel, but leave it open for the multis in GCYM?